### ISSUE TITLE
Apps not live reloading after saving. 

### ISSUE THREAD

**Issue by RealDeclan** [ironmansoftware/powershell-universal#3681]
### Version

5.0.4

### Severity

Medium

### Environment

Container

### Steps to Reproduce

1. Create an App
2. Add some code to dashboard - I used anything from https://docs.powershelluniversal.com/apps/building-dashboards
3. Save the dashboard
4. Press "View App"
5. When the tab opens containing the app, modify the dashboard code
6. Save the Modified Code
7. The preview page will refresh and show "Page Not Found"
8. Refreshing the URL will show "Page Not Found"

### Expected behavior

```console
When I create/update the code for any one of my pages, i should see the updates appear in the current window i use to preview the app.
```

### Actual behavior

```console
When I create/update the code for any one of my pages, the page returns "Page not Found" and i need to reopen the page via the "View Page" button in PowerShell universal to see any changes that have been made. Also navigating to the full URL returns "Page Not Found"
```

**Comment by adamdriscoll** [comment]
I can't reproduce this. I am running the latest docker container (5.0.4) on my desktop.

**Comment by RealDeclan** [comment]
To Reproduce:

  1.  Create a Simple Task, 6am
  2.  Once created Check the time on the newly created task and its not 6am (For me anyway)
  3.  Modify the newly created schedule to the correct time and press save.
  4.  The task will now disappear.



I discovered after that the task deletion is happening during the update. The script field isn’t auto populated so when I press save it just deletes. If I update the task then choose the same script, the task persists.

Kind regards,​​​​

Declan

The contents of this email message and any attachments are intended solely for the addressee(s) and may contain confidential and/or privileged information and may be legally protected from disclosure.

From: Adam Driscoll ***@***.***>
Sent: Monday, September 2, 2024 10:35 PM
To: ironmansoftware/powershell-universal ***@***.***>
Cc: Declan E Dunne ***@***.***>; Author ***@***.***>
Subject: [DIFFERENT REPLY-TO] Re: [ironmansoftware/powershell-universal] Apps not live reloading after saving. (Issue #3681)

This Message has a different reply to address and therefore may not be coming from who it appears to be. Reply-To-Address: ***@***.******@***.***>
________________________________
I can't reproduce this. I am running the latest docker container (5.0.4) on my desktop.

**Comment by adamdriscoll** [comment]
The schedule issue should be resolved in 5.0.5. I saw the script not populating properly as well.

**Comment by RealDeclan** [comment]
Nice! I just noticed the next update. You are motoring with them! Well done!
Kind regards,​​​​

Declan E Dunne
Project Engineer
E: ***@***.******@***.***>
|
W: Ortus.ie<https://eu.content.exclaimer.net?url=http%3A%2F%2Fortus.ie%2F&tenantid=elFd28uoEeyZfgBQ8ofJ_Q&templateid=acea38c4b107ef1196f5000d3aaa3a50&excomponentid=mPjkvBPwUqtzBQF8jyfY3n04l-nJaD6-_O23Lmrj9_A&excomponenttype=Link&signature=y3Hw8S7itMhgljo368Qgf9ZQKzBeWGzebD4O3ll1zi2tur-MEwEKE4JgT27gRVLpxZFdWe28LY41OdBGfp4XoQkUl3fbmoDEPmRO0dT8yzAUx6rljE_K_jc_kMKK3DwuNf5SY8HJZikPG2UBoSRvKgHksKExlh5C-0bPjO4rN5Fqg30dASTm1QLzPjsXPopyQ96arEuxaDaPxF6FhEunXux7sgVr9DSNe5av9ppg1dInUTgY62_SYfUX7ZEFhSHHpWrexD0nZniW6cOzGYSQU_cXhi57tSpp_p3weCj6FKemgwe0XmCEMuB21eKj4as0RL4Tc8wi3a27CZuhv6VrAw&v=1>

The contents of this email message and any attachments are intended solely for the addressee(s) and may contain confidential and/or privileged information and may be legally protected from disclosure.

[cid:1884b284-2f4c-4b14-a0a1-1307a78e43d4]<https://eu.content.exclaimer.net/?url=https%3A%2F%2Fwww.ek.co%2F&tenantid=elFd28uoEeyZfgBQ8ofJ_Q&templateid=acea38c4b107ef1196f5000d3aaa3a50&excomponentid=BQYmbBBnNdIiV-JaSCqtBevUxWE9O9v_kLyz8Jm56WQ&excomponenttype=Image&signature=XJPk9oYXAuXv3Aqzhne4ptt_8Dg3-NXdzHzgjaaRMaX-RUY74pODdgyP-7AKkJ5xZTY3yLRHdrMIw8RWKD_CcuZduyS_0ImcqNDo_5Hah3L8sziNNWUEdKDdCHml_3Z02gbT7sHDmFJQBCpOnZHSqTcXhkrcSdseiMuTDq6SVxoftVj7RA0cxAWATwTsMAxvKfc7j_AnVU7iauAzi_L3yG-Bf75QJ6tW_QsbVCw87oXhKZFq64pgVmXOf77QBjxWGSSeboP7ay711_4LRYAr2elAS2pPgtU_kEvkHFyJfP2jV7Al5If0WOZ1caSDmzxfn5VsU7W4JajQaGnAi8t0nA&v=1>
[cid:5d36397a-40fa-4fc7-a4f8-79c3bcb0587e]
________________________________
From: Adam Driscoll ***@***.***>
Sent: 03 September 2024 14:19
To: ironmansoftware/powershell-universal ***@***.***>
Cc: Declan E Dunne ***@***.***>; Author ***@***.***>
Subject: [DIFFERENT REPLY-TO] Re: [ironmansoftware/powershell-universal] Apps not live reloading after saving. (Issue #3681)

This Message has a different reply to address and therefore may not be coming from who it appears to be. Reply-To-Address: ***@***.******@***.***>
________________________________
The schedule issue should be resolved in 5.0.5. I saw the script not populating properly as well.
